models/user.js

```diff
@@ -18,7 +18,7 @@ const userSchema = new mongoose.Schema({
     },
     phoneNumber: {
         type: String,
-        required: [true, 'please provide a phone number'],
+        //required: [true, 'please provide a phone number'],
```

Comment:
phone number should be required i think ???? that is used to validate real user too